main.py

```diff
@@ -1,10 +1,55 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r", encoding="utf-8") as players_file:
+        players = json.load(players_file)
```

Comment:
The context manager is being overloaded. According to the checklist, you should exit the context manager block after reading the data from the file. Consider moving the code that processes the data outside of the 'with' block.